AWSTranscribeStreaming/AWSTranscribeStreamingModel.m

```diff
@@ -338,26 +658,90 @@ + (NSValueTransformer *)languageCodeJSONTransformer {
                 return @"ko-KR";
             case AWSTranscribeStreamingLanguageCodeZhCN:
                 return @"zh-CN";
-            case AWSTranscribeStreamingLanguageCodeHiIN:
-                return @"hi-IN";
             case AWSTranscribeStreamingLanguageCodeThTH:
                 return @"th-TH";
-            default:
-                return nil;
-        }
-    }];
-}
-
-+ (NSValueTransformer *)mediaEncodingJSONTransformer {
```

Comment:
For some weird reason the diff gets confused and thinks this method has been removed, when it's [totally still in there](https://github.com/aws-amplify/aws-sdk-ios/blob/ruisebas/transcribestreaming-languages/AWSTranscribeStreaming/AWSTranscribeStreamingModel.m#L434-L448) 🤷‍♂️